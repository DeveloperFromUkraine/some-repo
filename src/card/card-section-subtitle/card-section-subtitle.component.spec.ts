import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardSectionSubTitleComponent } from "index";
import { ComponentTest } from "../../../test/test-bed/component";


describe('CardSectionSubTitleComponent', () => {
    let fixture: ComponentFixture<CardSectionSubTitleComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardSectionSubTitleComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardSectionSubTitleComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});