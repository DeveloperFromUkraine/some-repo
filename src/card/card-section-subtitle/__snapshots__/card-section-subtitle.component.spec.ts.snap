// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSectionSubTitleComponent should match snapshot 1`] = `
<ign-card-section-subtitle>
  <h3
    class="ign-card-section-subtitle mat-card-subtitle"
  >
    
  </h3>
</ign-card-section-subtitle>
`;
